addons/l10n_tn/data/account_chart_template_data.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_tn_chart_template" model="account.chart.template">
+        <field name="property_account_receivable_id" ref="l10n_tn_4111"/>
+        <field name="property_account_payable_id" ref="l10n_tn_4011"/>
+        <field name="property_account_income_categ_id" ref="l10n_tn_7071"/>
+        <field name="property_account_expense_categ_id" ref="l10n_tn_607"/>
+        <field name="income_currency_exchange_account_id" ref="l10n_tn_756"/>
+        <field name="expense_currency_exchange_account_id" ref="l10n_tn_655"/>
+        <field name="default_pos_receivable_account_id" ref="l10n_tn_5411" />
+        <field name="property_tax_payable_account_id" ref="l10n_tn_43651"/>
+        <field name="property_tax_receivable_account_id" ref="l10n_tn_436711"/>
+        <field name="account_journal_early_pay_discount_loss_account_id" ref="l10n_tn_6091"/>
+        <field name="account_journal_early_pay_discount_gain_account_id" ref="l10n_tn_7091"/>
+        <field name="default_cash_difference_income_account_id" ref="l10n_tn_756"/>
+        <field name="default_cash_difference_expense_account_id" ref="l10n_tn_655"/>
+    </record>
+</odoo>
```

Comment:
Check your IDE/editor to include a EOL char on the last line at least in Python/XML/Javascript